fetch/api/cors/cors-preflight-star.any.js

```diff
@@ -16,7 +16,7 @@ function preflightTest(succeeds, withCredentials, allowMethod, allowHeader, useM
     if (useMethod) {
       requestInit.method = useMethod
     }
-    if (useHeader) {
+    if (useHeader.length > 0) {
```

Comment:
Can you tell me the intention of this change?

Reply:
It seemed better to use an empty array than undefined. Gives slightly better error messages and reads better. And I changed this because I wasn't sure if an empty array would be false.